chainer/function_hooks/cuda_profile.py

```diff
@@ -0,0 +1,25 @@
+from chainer import cuda
+from chainer import function
+
+if cuda.available:
+    from cupy.cuda import nvtx
```

Comment:
Hum. Same problem exists in other modules. How about making a helper method or something like that?